tests/arp/test_arp_packet_e2e.py

```diff
@@ -81,11 +81,11 @@ def captures_ok(api, cfg, size, utils):
         [0x00, 0x0C, 0x30, 0xE3, 0x50, 0xEA],
     ]
     sender_protocol_addr = [
-        [0x0a, 0x01, 0x01, 0x02],
-        [0x0a, 0x01, 0x01, 0x03],
-        [0x0a, 0x01, 0x01, 0x04],
-        [0x0a, 0x01, 0x01, 0x05],
-        [0x0a, 0x01, 0x01, 0x06],
+        [0x0A, 0x01, 0x01, 0x02],
```

Comment:
not sure why dpkt is changing case. is it not possible to handle this thing in utility level (`get_all_captures`) using dpkt option.
Interestingly Athena test repo dpkt able to accept both case. 
..\tests\py\gre\test_list_gre_fields_with_checksum_flag.py
..\tests\py\gtpv1\test_all_counter_gtpv1_teid.py

Reply:
Done.